pages/common/pulumi-version.md

```diff
@@ -0,0 +1,12 @@
+# pulumi version
+
+> Print Pulumi’s version number.
+> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_version/>.
+
+- Print Pulumi's version number:
+
+`pulumi version`
+
+- Display help:
+
+`pulumi version {{-h|--help}}`
```

Comment:
```suggestion
# pulumi version

> Display Pulumi’s version number.
> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_version/>.

- Display version:

`pulumi version`

- Display help:

`pulumi version {{-h|--help}}`
```
we use display here too, like in the help example, see styleguide

Reply:
I should know better :)